### ISSUE TITLE
Fix pyflakes for lore

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6998]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#6998|
|Type|enhancement|
|Created|2014-02-27 13:08:35Z|
|Branch|https://github.com/twisted/twisted/tree/pyflakes-lore-6998|

Attachments:

* [6998-1.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/788/7885a5416019fe7fd17d4b3522cd44781d1d7448/2a9480ae390c31315e916cfe7f10e8aa7138ae4a.diff) (2358 bytes) - added by adiroiban on 2014-02-27 13:14:03Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6998 6998
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__ 
branch__branches_pyflakes_lore_6998 branches/pyflakes-lore-6998
branch_author__rwall rwall
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__ 
time__1393506515000000 1393506515000000
changetime__1396962208000000 1396962208000000
version__None None
owner__adiroiban adiroiban

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban commented|
|-|-|

Just simple cleanup.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)|@wallrj set owner to @wallrj<br>@wallrj set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_3"></a>|@wallrj commented|
|-|-|

(In [42119]) Branching to 'pyflakes-lore-6998'

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_4"></a>|@wallrj commented|
|-|-|

(In [42120]) Apply 6998-1.diff from adiroban. Refs [#6998](https://github.com/twisted/twisted/issues/6998).

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_5"></a>|@wallrj removed owner<br>@wallrj set status to `new`|
|-|-|

Ok, there are some buildbot configuration problems since r42118. 
I'll finish reviewing when those are fixed.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)|@wallrj set owner to @wallrj<br>@wallrj set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_7"></a>|@wallrj set owner to @adiroiban<br>@wallrj set status to `new`|
|-|-|

Thanks adiroban,

Notes:
 1. Forced build: https://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/pyflakes-lore-6998
    1. There's one spurious debian6-x86_64-py2.6 which I haven't seen before

Points:
 1. [branches/pyflakes-lore-6998/twisted/lore/indexer.py](branches/pyflakes-lore-6998/twisted/lore/indexer.py)
    1. I'm not sure about "entries = None". The functions in indexer.py seem to
       expect this to be iterable. Perhaps it should be "entries = {}". But on
       the other hand, these functions can never have worked unless
       setIndexFilename or clearEntries had been called first so perhaps None is
       a good default and perhaps it doesn't matter since lore is about to
       deprecated.


I'll merge.

-RichardW.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_8"></a>|@wallrj set status to `closed`|
|-|-|

(In [42138]) Merge pyflakes-lore-6998

Author: adiroiban
Reviewer: rwall
Fixes: [#6998](https://github.com/twisted/twisted/issues/6998)

Cleanup unused imports and uninitialised variables in twisted.lore. Twisted
Lore is now "pyflakes clean".
